healthyregions/chicago-environment-explorer
# Fix Footer logo, clear various build/runtime warnings/errors

## Problem
Various build warning / innocuous errors in the dev console and/or build logs

Footer logo should be shifted to the right, and DePaul logo should be included in Footer

Fixes #94 

## Approach
* fix: About page - build warnings from links (missing `rel` attribute)
* fix: Map page - `<div> cannot be a child of <p>` error (Legend)
* fix: Community page - missing `key` prop in `.map()`
* feat: Footer - include DePaul logo, shift HeRoP logo to the right
    * NOTE: ~~I had trouble finding a greyscale logo for DePaul University, so I just reused our existing one with CSS [grayscale](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/grayscale) and [invert](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/invert) filters~~ Adam provided a grayscale image for us to use :+1:

![Screenshot 2024-06-26 at 5 56 54 PM](https://github.com/healthyregions/chicago-environment-explorer/assets/1413653/efefb57b-7815-4f58-84f6-8836c0bb9de7)


## How to Test
1. Navigate to https://deploy-preview-167--chicago-env-explorer.netlify.app
2. Scroll down to Footer
    * You should see DePaul logo is now shown here in the center of the Footer
    * You should see that clicking the DePaul logo links off to their website
    * You should see that the HeRoP logo now appears to the right in the Footer
